### ISSUE TITLE
Boolean method identified as trivial reader method

### ISSUE THREAD

**Issue by dkubb** [rubocop/rubocop#411]
I have a method like:

``` ruby
def verbose?
  @verbose
end
```

rubocop is warning me with "Use attr_reader to define trivial reader methods", which I don't think should apply. I don't want to rename the method to `#verbose` because I don't want to expose the internal state of the object.
